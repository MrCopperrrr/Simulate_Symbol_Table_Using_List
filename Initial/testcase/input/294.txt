BEGIN
INSERT v number
INSERT c string
LOOKUP v
LOOKUP c
INSERT m string
INSERT o number
LOOKUP m
LOOKUP o
INSERT z string
INSERT u string
LOOKUP z
LOOKUP u
INSERT n string
INSERT b string
LOOKUP n
LOOKUP b
INSERT x number
LOOKUP x
END