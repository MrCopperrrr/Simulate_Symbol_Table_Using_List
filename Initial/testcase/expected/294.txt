success
success
1
1
success
success
1
1
success
success
1
1
success
success
1
1
success
1